rpc/rpcserver/server.go

```diff
@@ -277,14 +279,52 @@ func (s *walletServer) ImportPrivateKey(ctx context.Context, req *pb.ImportPriva
 			"Only the imported account accepts private key imports")
 	}
 
-	_, err = s.wallet.ImportPrivateKey(wif, nil, req.Rescan, 0)
+	if req.ScanFrom < 0 {
```

Comment:
Good check but I believe it would also be a good idea to error when scan_from is non-zero and rescan is false.